src/porepy/models/material_constants.py

```diff
@@ -348,6 +349,8 @@ def default_constants(self) -> dict[str, number]:
             "thermal_conductivity": 1,
             "thermal_expansion": 0,
             "well_radius": 0.1,
+            "tangential_characteristic_tol": 1e-5,  # Numerical method parameter
+            "contact_mechanics_scaler": 1e-1,  # Numerical method parameter
```

Comment:
```suggestion
            "contact_mechanics_scalar": 1e-1,  # Numerical method parameter
```
This applies to the method name below as well.

Reply:
Also, can we come up with better names? I can guess on the meaning of tangential_characteristic_tol, but not of contact_mechanics_scalar

Reply:
Super happy to see possibility to control these parameters externally! And I partly do not necessarily agree with you Eirik: I can guess the meaning of tangential_characteristic_tol, but I am not sure it is the right one from physical point of view. To me the associated characteristic function determines whether contact is open or closed. Its relation to "tangential" is only through the use in the tangential fracture deformation.